seq2simulate/run_make_proviral_data.py

```diff
@@ -393,10 +436,10 @@ def _parse_sam_line(read_id, sam_file, paired_end=False):
         elif int(sam_file[read_id][0][FLAG]) == int(sam_file[read_id][1][FLAG]):
             raise ValueError('Both paired-end FASTQ reads are in the same direction.')
         
-        read_start_f = int(sam_file[read_id][0][POS])
-        read_start_r = int(sam_file[read_id][1][POS])
-        read_end_f = read_start_f+len(sam_file[read_id][0][SEQ])-1
-        read_end_r = read_start_r+len(sam_file[read_id][1][SEQ])-1
+        read_start_f = int(sam_file[read_id][0][POS]) - 1 # convert to 0-based
+        read_start_r = int(sam_file[read_id][1][POS]) - 1 # convert to 0-based
+        read_end_f = read_start_f+len(sam_file[read_id][0][SEQ]) - 1
+        read_end_r = read_start_r+len(sam_file[read_id][1][SEQ]) - 1
```

Comment:
As below, not sure if the end needs changing because of now zero start offset, or if this is corrected now?